pkg/otlp/logs/translator.go

```diff
@@ -65,14 +65,14 @@ func (t *Translator) hostFromAttributes(ctx context.Context, attrs pcommon.Map)
 }
 
 // MapLogs from OTLP format to Datadog format.
-func (t *Translator) MapLogs(ctx context.Context, ld plog.Logs) []datadogV2.HTTPLogItem {
+func (t *Translator) MapLogs(ctx context.Context, ld plog.Logs, hostFromAttributesHandler attributes.HostFromAttributesHandler) []datadogV2.HTTPLogItem {
```

Comment:
This will lead to another circular dependency problem between Agent and collector. Does it make sense to add a new func like `MapLogsWithHost` while leaving `MapLogs` as-is?

Reply:
There is no circular dependency (at least I didn't spot any), but indeed I have to update `datadog-agent` (PR is nearly ready) and `opentelemetry-collector-contrib` (PR is ready and is a one line change).
Let me know if you still think it is better to add `MapLogsWithHost` and remove it latter to reduce friction.

Reply:
Got it, for this one it's less of a problem, you might still need to do update version in Agent -> merge PR -> use a commit hash version of Agent in Collector.